Lib/test/test_pyexpat.py

```diff
@@ -771,6 +771,22 @@ def resolve_entity(context, base, system_id, public_id):
         self.assertEqual(handler_call_args, [("bar", "baz")])
 
 
+class ParentParserLifetimeTest(unittest.TestCase):
+    """
+    Subparsers make use of their parent XML_Parser inside of Expat.
+    As a result, parent parsers need to outlive subparsers.
+
+    See https://github.com/python/cpython/issues/139400.
+    """
+    def test_parent_parser_outlives_its_subparsers(self):
```

Comment:
```suggestion
    """

    def test_parent_parser_outlives_its_subparsers(self):
```

Just for readability of the docstring.

Reply:
@picnixz can do, upcoming…